src/places.py

```diff
@@ -384,3 +401,41 @@ def _on_encrypted_eject_finished(self, encrypted, success):
             self._on_device_removed(None, encrypted)
         else:
             self.emit("failed", None)
+
+    def _find_place_by_path(self, listbox, path):
+        for place in listbox:
+            if place.path == path:
+                return place
+        return None
+
+    def _remove_bookmark(self, bookmark, path):
```

Comment:
Please follow the same naming conventions for other callbacks (e.g., _on_bookmark_removed).